src/app.js

```diff
@@ -7,7 +7,7 @@ import './util/polyfills'
 import { Client } from 'bugsnag-react-native'
 import React, { Component } from 'react'
 import { AppState, AsyncStorage, Platform, Text, TextInput } from 'react-native'
-import BackgroundTask from 'react-native-background-task'
+import BackgroundFetch from 'react-native-background-fetch'
```

Comment:
why did this change?

Reply:
Because the old library wasn't maintained / compatible with the new RN.